### ISSUE TITLE
Option to disable Xbox

### ISSUE THREAD

**Issue by UDPSendToFailed** [GeyserMC/Geyser#1198]
I'm using KernelPanic nickname on Java Edition servers, but it's impossible to change my Xbox gamer tag to KernelPanic because it's already taken. 
Doesn't matter what I do, it gives me some random nickname, not the one I need to play on the servers. I have tried some cracked Bedrock Edition APKs, but they aren't working because of the Xbox requirement, which forces me to use another nickname than the one I need.

![kép](https://user-images.githubusercontent.com/16057404/91156236-311a3100-e6c4-11ea-9b79-d1b7d045223f.png)

TL;DR: Xbox makes it impossible to play with my nickname and there is no option to disable it.

**What feature do you want?**

Config option to disable Xbox requirement.

**Comment by Tim203** [comment]
There is a plugin system for Geyser in the making and it has a plugin called GeyserLogin, that plugin does exactly what you want: https://bundabrg.github.io/GeyserLogin/

**Comment by UDPSendToFailed** [comment]
Thanks for the idea! I have installed the plugin but it doesn't seem to work. 

![kép](https://user-images.githubusercontent.com/16057404/91160366-991f4600-e6c9-11ea-9803-138884caed73.png)

**Comment by bundabrg** [comment]
I'll have a look at that. Been having some issues with dependencies when geyer is non standalone.

**Comment by DoctorMacc** [comment]
Geyser does not support piracy.

Another option is to enable online mode in Geyser and input your desired username into the login field. Other than that and the GeyserLogin plugin you're on your own.
